stage1/rootfs/usr_from_host/Makefile

```diff
@@ -1,13 +1,19 @@
 # do not include systemd in stage1, let's find it at run-time
 
-usr.done install: Makefile
+usr.done: Makefile install
+	cp -f install ../aggregate/install.d/00usr
+	touch usr.done
```

Comment:
Because the install target is phony, usr.done will always run. I don't really see the point in touching usr.done. It's going to be ignored by make anyway.